aws/client/client.go

```diff
@@ -89,7 +89,10 @@ const logReqMsg = `DEBUG: Request %s/%s Details:
 
```

Comment:
Sure, I've added this formatting in a766cba. I just removed the word `Details` because it doesn't actually log any request/response details.